google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ExternalTableDefinition.java

```diff
@@ -124,6 +124,12 @@ public Builder setFormatOptions(FormatOptions formatOptions) {
      */
     public abstract Builder setCompression(String compression);
 
+    /**
+     * [Optional, Trusted Tester] Connection for external data source. The value may be {@code
```

Comment:
Consider documenting the format of the connectionId string in the comments.

Reply:
done. thank you!